### PR TITLE
feat: add feeds schema

### PR DESCRIPTION
**Pull request by chandankumar1307**
Added a Post Schema containing these feeds:
Title,
message,
likes,
comments,
tags

### CODE REVIEW

File: server/Controllers/feeds.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  const newFeed = new Feeds(feeds);
+  console.log(feeds);
+
+  try {
+    await newFeed.save();
+    res.status(201).json(newFeed);
+  } catch (error) {
+    res.status(409).json({ message: error.message });
+  }
+};
```
Comment (SuperAayush) [inline server/Controllers/feeds.js:24]: Add a bunch of comments to make it user-friendly.

File: server/Schema/feeds.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import mongoose from "mongoose";
+
+const FeedSchema = new mongoose.Schema({
+  userName: {
+    type: String,
+    required: true,
+  },
+  dp: {
+    type: String,
+    // required: true,
```
Comment (SuperAayush) [inline server/Schema/feeds.js:10]: Better remove the commented code before pushing.

**Review by SuperAayush — CHANGES_REQUESTED** [review]

File: server/Controllers/feeds.js (hunk 1 above)
Comment (chandankumar1307) [reply server/Controllers/feeds.js:26]: Added a few comments to describe the code.

File: server/Schema/feeds.js (hunk 2 above)
Comment (chandankumar1307) [reply server/Schema/feeds.js]: Removed commented codes

**Review by SuperAayush — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**